HW_01/hw_1_05.py

```diff
@@ -0,0 +1,12 @@
+# Напишите программу, которая принимает на вход координаты двух точек и находит расстояние между ними в 2D пространстве.
+# Пример:
+# - A (3,6); B (2,1) -> 5,09
+# - A (7,-5); B (1,-1) -> 7,21
+
+import math
+x1 = int(input('Введите координату X точки A: '))
+y1 = int(input('Введите координату Y точки A: '))
+x2 = int(input('Введите координату X точки B: '))
+y2 = int(input('Введите координату Y точки B: '))
+print('Расстояние между заданными точками в 2D пространстве = ',
+      round(math.sqrt((abs(x1-x2))**2 + (abs(y1-y2))**2), 2))
```

Comment:
Можно вполне обойтись без библиотеки `math`. Функция `abs` также не нужна, так как из школьного курса математики вы знаете, что квадрат любого числа всегда положителен.
```suggestion
      round(((x1 - x2) ** 2 + (y1 - y2) ** 2) ** (1 / 2), 2))
```